modules/importer/data-importer.js

```diff
@@ -189,14 +190,14 @@ export default class DataImporter extends FormApplication {
           promises.push(OggDude.Import.Armor(xmlDoc, zip));
           promises.push(OggDude.Import.Talents(xmlDoc, zip));
           promises.push(OggDude.Import.ForcePowers(xmlDoc, zip));
-          promises.push(OggDude.Import.SignatureAbilities(xmlDoc, zip));
+          //promises.push(OggDude.Import.SignatureAbilities(xmlDoc, zip));
```

Comment:
Is this meant to be commented out?

Reply:
It _should_ be removed, but I don't think it does a ton either way. It's been re-added further down the file (line 222)